blog/hosted-service-v2

```diff
@@ -0,0 +1,66 @@
+
+# Exciting News: V2 Hosted Service is Here! 🚀
+[insert blog asset]
+
+Hold on to your Moonlanders—V2 Hosted Service has arrived! It’s a significant upgrade that brings you faster build times, enhanced features, and an overall smoother experience. Starting **Wednesday, Nov 6th, 2024**, all new deployments will automatically utilize V2. Let’s see what you can expect!👇
+
+
+## What’s New with V2?
+
+Here’s what you can expect from the V2 Hosted Service:
+
+
+
+* 10x faster build and deployment times 
+* Faster indexing speeds
+* Improved UI/UX
+
+More features:
+
+
+
+* Access to direct database connections, IP address whitelisting, advanced analytics, and more!
+
+Behind-the-scenes enhancements: 
+
+
+
+* Experience increased flexibility, improved reliability, and reduced costs.
+
+*🚨Please note: From November 6th, 2024, all new indexer deployments will automatically route to the V2 Hosted Service instead of the current V1 Hosted Service. If you have existing indexers, any new pushes will deploy to V2 as well. *
+
+You can explore your indexers in our new [Explorer](https://v2.hyperindex.xyz/), while your V1 indexers will still be accessible at[ envio.dev](http://envio.dev).
```

Comment:
https://v2.hyperindex.xyz/ 

Please hold off on publishing this till this is clarified

Reply:
https://v2.envio.dev/explorer

Reply:
Thanks @DenhamPreen & @DZakh! I've updated the explorer link. 😄